.travis.yml

```diff
@@ -76,6 +76,7 @@ before_install:
 
 install:
   - ccache -s
+  - git describe
```

Comment:
I'm not concerned about this, but would like to make sure that you intended to have this here.